NuGet.Docs/_Layout.cshtml

```diff
@@ -1,104 +1,111 @@
-                <li><a href="//www.nuget.org/packages">Packages</a></li>
-                <li class="current"><a href="~/">Documentation</a></li>
-                <li><a href="https://www.nuget.org/downloads">Downloads</a></li>
-                <li><a href="http://blog.nuget.org">Blog</a></li>
-                <li><a href="//www.nuget.org/policies/contact">Contact</a></li>
-            </ul>
-        </nav>
-        <div style="clear:both"></div>
-    </div>
-    <div class="subnav">
-        <nav class="sub">
-            <ul>
-                <li id="subnav-consume"><a href="~/consume">Consume</a></li>
-                <li id="subnav-create"><a href="~/create">Create</a></li>
-                <li id="subnav-contribute"><a href="~/contribute">Contribute</a></li>
-                <li id="subnav-releaseNotes"><a href="~/release-notes">Release Notes</a></li>
-            </ul>
-        </nav>
-        <div class="twitter">
-            <a href="https://twitter.com/nuget" class="twitter-follow-button" data-show-count="false" data-dnt="true">Follow @@nuget</a>
-            <script>!function (d, s, id) { var js, fjs = d.getElementsByTagName(s)[0], p = /^http:/.test(d.location) ? 'http' : 'https'; if (!d.getElementById(id)) { js = d.createElement(s); js.id = id; js.src = p + '://platform.twitter.com/widgets.js'; fjs.parentNode.insertBefore(js, fjs); } }(document, 'script', 'twitter-wjs');</script>
-        </div>
-        <div style="clear:both"></div>
-    </div>
-    <div class="content">
-        <section>
+        <div id="page-content-wrapper">
             @RenderBody()
-            <div style="clear:both;"></div>
-        </section>
+        </div>
     </div>
-    <footer class="page">
-        <p>
-            <a href="http://www.dotnetfoundation.org"><img src="~/images/dnf.png" alt=".NET Foundation" style="border-width:0" /></a>
-        </p>
-        <p>
-            &copy; 2011-@DateTime.UtcNow.Year .NET Foundation - <a href="//www.nuget.org/policies/terms">Terms of Use</a> - <a href="//www.nuget.org/policies/privacy">Privacy Policy</a> - <a href="http://status.nuget.org">Service status</a>
-        </p>
-        @if (Page.Source != null)
-        {
-            <p>
-                View or edit this page's source on GitHub:
-                <a href="https://github.com/NuGet/NuGetDocs/tree/master/NuGet.Docs/@HttpUtility.UrlPathEncode(Page.Source)" target="edit">https://github.com/NuGet/NuGetDocs/tree/master/NuGet.Docs/@HttpUtility.UrlPathEncode(Page.Source)</a>
-            </p>
-        }
-        else
-        {
-            <p>
-                View or edit this site's source on GitHub:
-                <a href="https://github.com/NuGet/NuGetDocs" target="edit">https://github.com/NuGet/NuGetDocs</a>
-            </p>
-        }
-        @RenderSection("footer", required: false)
-    </footer>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
```

Comment:
The project contains these versions of jQuery:
* 1.9.1
* 1.10.0

And then here we reference 1.12.2 from an external location. Which is the one that is effectively in use? And can we get rid of the unused versions in the docs site to avoid any confusion/versioning issues?